src/test/java/somethingrandom/dataaccess/local/AddItemTest.java

```diff
@@ -0,0 +1,126 @@
+        item1Data.put("neededTime", item1.getNeededTime().toString());
+
+        item2Data.put("name", item2.getName());
+        item2Data.put("id", item2.getID().toString());
+        item2Data.put("creationDate", item2.getCreationDate().toString());
+        item2Data.put("itemKind", item2.getItemKind());
+        item2Data.put("remindDate", item2.getReminderDate().toString());
+
+        item3Data.put("name", item3.getName());
+        item3Data.put("id", item3.getID().toString());
+        item3Data.put("creationDate", item3.getCreationDate().toString());
+        item3Data.put("itemKind", item3.getItemKind());
+        item3Data.put("description", item3.getDescription());
+
+        try {
+            FileUserDataAccessObject dao = new FileUserDataAccessObject("./data.json");
+            dao.save(item1);
+            dao.save(item2);
+            dao.save(item3);
+        } catch (DataAccessException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    /**
+     *
+     * Test that the items are saved to the file as expected.
+     */
+
+    @org.junit.Test
+    public void testActionableItemSavedProperly() {
+        try{
+            BufferedReader reader = new BufferedReader(new FileReader("./data.json"));
+            ObjectMapper mapper = new ObjectMapper();
+            JsonNode parser = mapper.readTree(reader);
+
+            assert item1Data.get("name").equals(parser.path(item1Data.get("id")).path("name").asText());
+            assert item1Data.get("id").equals(parser.path(item1Data.get("id")).path("id").asText());
+            assert item1Data.get("creationDate").equals(parser.path(item1Data.get("id")).path("creationDate").asText());
+            assert item1Data.get("itemKind").equals(parser.path(item1Data.get("id")).path("itemKind").asText());
+            assert item1Data.get("neededTime").equals(parser.path(item1Data.get("id")).path("neededTime").asText());
+
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @org.junit.Test
+    public void testDelayedItemSavedProperly() {
+        try{
+            BufferedReader reader = new BufferedReader(new FileReader("./data.json"));
+            ObjectMapper mapper = new ObjectMapper();
+            JsonNode parser = mapper.readTree(reader);
+
+            assert item2Data.get("name").equals(parser.path(item2Data.get("id")).path("name").asText());
+            assert item2Data.get("id").equals(parser.path(item2Data.get("id")).path("id").asText());
+            assert item2Data.get("creationDate").equals(parser.path(item2Data.get("id")).path("creationDate").asText());
+            assert item2Data.get("itemKind").equals(parser.path(item2Data.get("id")).path("itemKind").asText());
+            assert item2Data.get("remindDate").equals(parser.path(item2Data.get("id")).path("remindDate").asText());
+
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
For all your tests: you can just `throws IOException`, JUnit should deal with it.